src/main/java/org/drugis/addis/problems/service/impl/SingleStudyBenefitRiskServiceImpl.java

```diff
@@ -58,13 +61,16 @@
   @Inject
   private AbsoluteStudyBenefitRiskService absoluteStudyBenefitRiskService;
 
+  @Inject
+  private VersionMappingRepository versionMappingRepository;
+
   @Override
   public SingleStudyBenefitRiskProblem getSingleStudyBenefitRiskProblem(
           Project project,
           BenefitRiskStudyOutcomeInclusion inclusion,
           Outcome outcome,
           Set<AbstractIntervention> includedInterventions
-  ) {
+  ) throws IOException {
```

Comment:
as said in previous comment, I don't think letting minor exceptions leak out & crash the whole flow is the best solution